PyDynamic/uncertainty/propagate_filter.py

```diff
@@ -39,6 +39,8 @@ def FIRuncFilter(y, sigma_noise, theta, Utheta=None, shift=0, blow=None, kind="c
             FIR filter coefficients
         Utheta: np.ndarray
             covariance matrix associated with theta
+            if the filter is fully certain, use `Utheta = None` to make use of more efficient calculations. 
+            see also the comparison given in <examples\Digital filtering\FIRuncFilter_runtime_comparison.py>
         shift: int
             time delay of filter output signal (in samples)
         blow: np.ndarray
```

Comment:
We could (/should?) mention the defaults.
```suggestion
        Utheta: np.ndarray, optional
            covariance matrix associated with theta
            if the filter is fully certain, use `Utheta = None` (default) to make use of more efficient calculations. 
            see also the comparison given in <examples\Digital filtering\FIRuncFilter_runtime_comparison.py>
        shift: int, optional
            time delay of filter output signal (in samples) (defaults to 0)
        blow: np.ndarray, optional
```